coalib/core/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Provides an API for coala's NextGen-Core
+
+- ``Bear``: Base class for all the NextGen Bears.
+- ``Core``: Maintains sessions for coala's execution.
```

Comment:
As a module docstring doesnt need to be a listing of components, these should be ordered to help the reader understand the module.

And more *prose* used to help the reader understand what it is.

I feel `Core` should be first, **or** the description of `Core` should mention it is about execution of *bears*.

is Core about tasks or bears?

If bears are just a type of task, and other tasks could also be performed using Core, then the concept of a task should be mentioned in *prose* before mentioning any bears.

Reply:
It's both, it runs tasks offloaded by bears